src/iptvsimple/Settings.h

```diff
@@ -129,38 +129,6 @@ namespace iptvsimple
     Settings(Settings const&) = delete;
     void operator=(Settings const&) = delete;
 
```

Comment:
After the set setting is fixed lets see what it looks like. The purpose this template was to tidy of the setting code and to always log setting changes.

Reply:
Can bring back a template to to log them, NP.